### PR TITLE
dotCMS/core#16792 Add friendly message when there are no rules

### PR DESCRIPTION
**Pull request by hmoreras**

### CODE REVIEW

File: projects/dot-rules/src/lib/rule-engine.ts [hunk 1]
```diff
@@ -46,7 +46,8 @@ const I8N_BASE = 'api.sites.ruleengine';
     </div>
   </div>
   <div class="cw-rule-engine__empty" *ngIf="!rules.length">
-      <span>{{rsrc('inputs.want.to.add.rules') | async}}</span>
+      <i class="material-icons">tune</i>
```
Comment (fmontes) [inline projects/dot-rules/src/lib/rule-engine.ts]: We have a dot-icon component: https://github.com/dotCMS/core-web/blob/master/src/app/view/components/_common/dot-icon/dot-icon.component.html#L1

File: projects/dot-rules/src/lib/styles/rule-engine.scss [hunk 2]
```diff
@@ -505,9 +507,23 @@ $finder-s-input: '.ui.input input';
...
+        i {
+            font-size: 120px;
+            color: $gray-light;
+            border: 7px solid;
+            border-radius: 10px;
+            margin-bottom: $basic-padding * 3;
+        }
+
+        h2 {
+            font-size: 22px;
```
Comment (fmontes) [inline projects/dot-rules/src/lib/styles/rule-engine.scss]: We have variables for this.

File: projects/dot-rules/src/lib/styles/rule-engine.scss [hunk 3]
```diff
@@ -14,6 +14,8 @@ $foreground-active: #848484;
 $text-emphasis: #fc7e1c;
 $warn: #cb3927;
 $brand-primary-one: #209021;
+$gray: #7e7a86;
```
Comment (fmontes) [inline projects/dot-rules/src/lib/styles/rule-engine.scss]: Are this the same grays we have in the app variables?

Comment (hmoreras) [reply projects/dot-rules/src/lib/styles/rule-engine.scss]: yes

**Pull request merged**